playbooks/common/openshift-cluster/update_repos_and_packages.yml

```diff
@@ -8,6 +8,5 @@
           ansible_distribution == "RedHat" and
           lookup('oo_option', 'rhel_skip_subscription') | default(rhsub_skip, True) |
             default('no', True) | lower in ['no', 'false']
```

Comment:
Looking at the test for rhel_skip_subscription, we should probably update it while we are in here:
```
     when: deployment_type == "enterprise" and
            ansible_distribution == "RedHat" and
            lookup('oo_option', 'rhel_skip_subscription') | default(rhsub_skip, True) |
              default('no', True) | lower in ['no', 'false']
```
should probably be:
```
     when: deployment_type in ["enterprise", "openshift-enterprise", "atomic-enterprise"] and
            ansible_distribution == "RedHat" and
            not (lookup('oo_option', 'rhel_skip_subscription') | default(rhsub_skip, True) | bool)
```